ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -555,6 +555,16 @@ static TimeDuration leaderStepDownWaitTime(RaftProperties properties) {
     static void setLeaderStepDownWaitTime(RaftProperties properties, TimeDuration noLeaderTimeout) {
       setTimeDuration(properties::setTimeDuration, LEADER_STEP_DOWN_WAIT_TIME_KEY, noLeaderTimeout);
     }
+
+    String PRE_VOTE_KEY = PREFIX + ".pre-vote";
+    boolean PRE_VOTE_DEFAULT = true;
```

Comment:
Nit: might be better to make this default value as `false` so downstream applications (e.g. Ozone) won't feel surprising before benchmarking this optimization.

Reply:
If we use false, when we change to true ? I think default true maybe better.

Reply:
@szetszwo What do you think ?

Reply:
I do not have a strong opinion.  It seems okay to set true as the default.

Reply:
Sure we can start from `true` then